README.md

```diff
@@ -1,21 +1,18 @@
 # conjur-api-python3
 
-Python3-based API SDK for [Conjur OSS](https://www.conjur.org/). The repo
-also includes a self-contained CLI tool (`conjur-cli`) that wraps the API
+Command-line interface and programmatic Python access to the Conjur API.
+
+The repo also includes a self-contained CLI tool (`conjur-cli`) that wraps the API
 in a simple executable script/binary.
```

Comment:
@izgeri do you know what does this mean?

Reply:
can you say which part is not clear? it's hard to tell what you're referring to from this comment

Reply:
"that wraps the API in a simple executable script/binary."
The executable script/binary part is not clear. Like the client wraps the API but as a library that can be imported so I don't understand what it means "wraps the API in a simple executable script/binary"

Reply:
I believe it's saying that the CLI is a wrapper around the Python client, and the CLI is an executable script/binary